demisto_sdk/commands/common/constants.py

```diff
@@ -915,25 +922,45 @@ def __init__(self, repo_name: Optional[str] = None):
         else:
             self.CURRENT_REPOSITORY = repo_name
         # DO NOT USE os.path.join on URLs, it may cause errors
-        self.CONTENT_GITHUB_LINK = urljoin(self.BASE_RAW_GITHUB_LINK, self.CURRENT_REPOSITORY)
-        self.CONTENT_GITHUB_MASTER_LINK = urljoin(self.CONTENT_GITHUB_LINK, r'master')
-        self.Credentials = GithubCredentials()
+        if not self.IS_GITLAB:
+            self.CONTENT_GITHUB_LINK = urljoin(self.BASE_RAW_GITHUB_LINK, self.CURRENT_REPOSITORY)
+            self.CONTENT_GITHUB_MASTER_LINK = urljoin(self.CONTENT_GITHUB_LINK, r'master')
+        else:
+            self.BASE_RAW_GITLAB_LINK = self.BASE_RAW_GITLAB_LINK.format(GITLAB_ID=self.GITLAB_ID)
 
-    @staticmethod
-    def _get_repository_name(urls: Iterable) -> str:
+    # @staticmethod
```

Comment:
make sure this doesn't break BC, and remove the comment afterward.
Are there any tests that use this method?

Reply:
There are tests, but they don't use this as static method.
I will add tests for getting repository name from gitlab

Reply:
added tests